src/rosdiscover/cli.py

```diff
@@ -48,11 +50,37 @@ def launch(args) -> None:
 
 
 def generate_acme(args):
+    with open(args.il.name, 'r') as f: 
+        data = yaml.load(f, Loader=yaml.FullLoader) #generates dictionary from YAML
```

Comment:
Prefer `yaml.safe_load(f)` rather than using `yaml.FullLoader`.

Reply:
Also, it's fine to drop the comment at the end of the line.

Reply:
I think you also need to check whether the argument is used before you try to open the file associated with it.